engo.go

```diff
@@ -118,6 +116,16 @@ func Run(o RunOptions, defaultScene Scene) {
 
 	opts = o
 
+	// TODO: We may need to check this later on for platform
+	InitGl()
```

Comment:
This is totally unnecessary, there's a reason why go has an [init function](https://golang.org/doc/effective_go.html#init).